src/Boards/Board.php

```diff
@@ -27,91 +27,42 @@ public function __construct($_width, $_height)
         {
             for ($x = 0; $x < $_width + 2; $x++)
             {
-                $this->grid[$x][$y] = 0;
+                $this->grid[$x][$y] = new Field($this, $x - 1, $y - 1);

```

Comment:
Now would be a good time to explain why your board is bigger than it should. :-)
For example in the documentation line above the loop. (Could not add a comment there because this line was not changed)